src/types/signature.rs

```diff
@@ -32,7 +31,7 @@ impl<'de> Deserialize<'de> for Signature {
         impl<'de> Visitor<'de> for SignatureVisitor {
             type Value = Signature;
 
-            fn expecting(&self, formatter: &mut Formatter) -> fmt::Result {
+            fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
GH doesn't seem to support stacking, so ignore this change (it's done in different PR)